pygmt/src/text.py

```diff
@@ -177,13 +177,23 @@ def text_(
         )
     ):
         kwargs.update({"F": ""})
-    if angle is not None and isinstance(angle, (int, float, str)):
+
+    if angle is True:
+        kwargs["F"] += "+a"
+    elif isinstance(angle, (int, float, str)):
         kwargs["F"] += f"+a{str(angle)}"
```

Comment:
Just thinking ahead to resolve #483 (allowing list inputs to angle/list/justify), we could simply use `if angle` (returns True for list/int/float/str inputs) like so:

```suggestion
    if angle:
        kwargs["F"] += "+a"
        if isinstance(angle, (int, float, str)):
            kwargs["F"] += str(angle)
```

Reply:
Actually, I was planing to work on #483, but I realized that the `Figure.text()` function needs more refactoring, so I opened PR #1121.

Personally, I prefer to make the changes to `text.py` as small as possible, and will address #483 after #1121 is merged.

Reply:
Ok, as you wish then.